app/index.html

```diff
@@ -35,8 +35,8 @@
 	})(window.M);
 </script>
 
-{{content-for 'instant-globals'}}
-<script src="/__load/-/only=scripts/wikia.ext.instantGlobals,instantGlobalsOverride,abtesting,abtest" onload="onInstantGlobalsLoaded()" onerror="onInstantGlobalsError()" defer></script>
+{{content-for 'async-scripts'}}
```

Comment:
how about async-scripts-handlers
async-scrips - sounds to me like this will add some async scripts